spec/end_of_life/repository_integration_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "climate_control"
+require "ostruct"
+
+RSpec.describe EndOfLife::Repository do
+  describe "#fetch" do
+    it "fetches all 200 repositories from an account despite exceeding page size" do
+      repositories = VCR.use_cassette("many_repositories") do
```

Comment:
Looks like this is not using the VCR cassette. Make sure to commit the cassettes if you want CI to use them.

Be sure that the cassettes don't leak your GITHUB_TOKEN. If they do, we can just rely on mocks, instead.

Reply:
Ah!  Got it.  Amazing what a few hours of good sleep can do ...

It _is_ using the cassette, but the check for the presence of GITHUB_TOKEN happens way before the cassette is read.  The fix is to set the environment variable in the spec.

Thanks for the warning re. leaking real tokens.  I've taken care of this with the following configuration:

`config.filter_sensitive_data("REDACTED") { ENV["GITHUB_TOKEN"] }`